src/FwdState.cc

```diff
@@ -234,7 +234,7 @@ void
 FwdState::completed()
 {
     if (flags.forward_completed) {
-        debugs(17, DBG_IMPORTANT, HERE << "FwdState::completed called on a completed request! Bad!");
+        debugs(17, DBG_IMPORTANT, "FwdState::completed called on a completed request! Bad!");
```

Comment:
this touched DBG_IMPORTANT line is missing a WARNING/ERROR log prefix.

also the "FwdState::completed" needs to be replaced with MYNAME